src/common/film.c

```diff
@@ -200,7 +200,7 @@ int dt_film_new(dt_film_t *film, const char *directory)
                                 -1, &stmt, NULL);
     DT_DEBUG_SQLITE3_BIND_TEXT(stmt, 1, datetime, -1, SQLITE_STATIC);
     DT_DEBUG_SQLITE3_BIND_TEXT(stmt, 2, directory, -1, SQLITE_STATIC);
-    dt_pthread_mutex_lock(&darktable.db_insert);
+    dt_pthread_mutex_lock(&darktable.db_film);
```

Comment:
Why this lock is needed at all?

Reply:
Right. Probably someone feeling that database safety work is pending? :-)

Reply:
A lot of other more dangerous stuff is done unlocked (e.g. removing a image), but this on the contrary looks safe to me...